ethcore/src/miner/miner.rs

```diff
@@ -1646,4 +1666,19 @@ mod tests {
 
 		assert!(miner.is_currently_sealing());
 	}
+
+	#[test]
+	#[cfg(not(feature = "price-info"))]
```

Comment:
Would be good to have test for both cases.

Reply:
I think the features setting is irrelevant for this test, let's just remove this line.

Reply:
I think it was left over from earlier, but let me double check.